modules/persistent-storage-csi-efs-metrics-procedure-cli.adoc

```diff
@@ -0,0 +1,61 @@
+// Module included in the following assemblies:
+//
+// * storage/persistent_storage/persistent-storage-csi-aws-efs.adoc
+// 
+
+[id="efs-metrics-procedure-cli_{context}"]
+= Enabling usage metrics using the CLI
+
+To enable AWS EFS storage CSI usage metrics using the CLI:
+
+. Edit the `ClusterCSIDriver` YAML file by running the following command:
++
+[source, terminal]
+----
+$ oc edit clustercsidriver efs.csi.aws.com -o yaml
+----
```

Comment:
Can you get a sample of this file? Might make it more clear where to make the edits.